Sources/Danger/GitHubDSL.swift

```diff
@@ -256,7 +256,7 @@ extension GitHub {
 extension GitHub {
     public struct Repo: Decodable, Equatable {
         enum CodingKeys: String, CodingKey {
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 259;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)